services/gateway/app.js

```diff
@@ -100,9 +100,9 @@ tempApp.run().then(async () => {
 		mixins: [ApiService, SocketIOService],
 		name: 'gateway',
 		actions: {
+			ready() { return getReady(); },
 			spec(ctx) { return genDocs(ctx); },
 			status() { return getStatus(this.broker); },
-			ready() { return getReady(this.broker); },
 			isBlockchainIndexReady() { return getIndexStatus(); },
```

Comment:
Perhaps we should consider renaming `isBlockchainIndexReady` to `blockchainIndexStatus` and update the implementation accordingly (in a separate issue)? 🤔